tests/product/tests/data/process/eks_process_test_cases.py

```diff
@@ -261,26 +271,124 @@
     "3.2.11 Kubelet config featureGates.RotateKubeletServerCertificate default, eval passed": cis_eks_3_2_11_pass,
 }
 
+eks_process_config_1 = {
+    **cis_eks_3_2_1_config_1,
+    **cis_eks_3_2_2_config_1,
+    **cis_eks_3_2_3_config_1,
+    **dict(
```

Comment:
same goes here, is there an alternative for zip + unpacking?

Reply:
As I mentioned before, it will be fixed in [this task](https://github.com/elastic/security-team/issues/5758)